_assets/compose/mailserver/Makefile

```diff
@@ -0,0 +1,65 @@
+GIT_ROOT = $(shell git rev-parse --show-toplevel)
+
+RED    := $(shell tput -Txterm setaf 1)
+GREEN  := $(shell tput -Txterm setaf 2)
+YELLOW := $(shell tput -Txterm setaf 3)
+RESET  := $(shell tput -Txterm sgr0)
+BOLD   := $(shell tput bold)
+
+# Settings
+export LOG_LEVEL = INFO
+export LISTEN_PORT ?= 30303
+export CONTAINER_NAME ?= status-go-mailserver
+export DATA_PATH ?= /var/tmp/status-go-mail
+# Necessary to make mailserver available publicly
+export PUBLIC_IP ?= $(shell curl -s https://ipecho.net/plain)
+
+FLEETS_URL := https://fleets.status.im/
+FLEET_NAME := eth.beta
+FLEETS = $(shell curl -s "$(FLEETS_URL)")
+BOOT_NODES = $(shell echo '$(FLEETS)' | jq '.fleets["$(FLEET_NAME)"].boot | map(.)')
+
+all: checks start show info enode
+
+checks:
+ifeq (, $(shell which docker-compose))
+	$(error No $(BOLD)docker-compoase$(RESET) in your $$PATH. Please install it)
+endif
+ifeq (, $(shell which jq))
```

Comment:
Isn't it going to return an error when running `make checks` because of `jq` used in `BOOT_NODES`?

Reply:
You're right, I forgot to remove that part since we do not update config anyway.